### ISSUE TITLE
Core dump on node add

### ISSUE THREAD

**Issue by benoit-canet** [scylladb/scylla#839]
In my experimental no timeout branch of scylla-longevity.

While running the node add test.

After initialization of the cluster and init of the small grape of node node are added.

At the second adding the test script wait forever for the new scylla node to be ready.

Once connected to the new node in SSH some initial compaction phase has succeded
and scylla is dead with coredump written on the disk.

gdb segfault on the target. (forget to do free on the target but the core dump is around 550 MB and the instance is micro type maybe
is an OOM like error).

Uploaded on:

scylladb-users-upload.s3.amazonaws.com/$report_uuid/issue#839.tar

Sorry forgot to compress

**Comment by slivne** [comment]
If its an OOM killer then in some cases in the past there was an indication for that in the the journalctl

**Comment by avikivity** [comment]
Am I supposed to guess $report_uuid?

You need to replace gdb by the one in our repository.

**Comment by benoit-canet** [comment]
Sorry =4d04af7c-85b9-4746-b8ea-e72ae0f4d10a

**Comment by benoit-canet** [comment]
@slivne: No OOM message in dmesg

**Comment by avikivity** [comment]
Acccess Denied

**Comment by avikivity** [comment]
@tzach ?

**Comment by avikivity** [comment]
oh, got it

**Comment by avikivity** [comment]
```
#0  0x00000000004c76f3 in std::__atomic_base<unsigned long>::fetch_or (__m=std::memory_order_relaxed, __i=32768, this=<error reading variable: Asked for position 0 of stack, stack only has 0 elements on it.>)
    at /opt/scylladb/include/c++/5.1.1/bits/atomic_base.h:544
#1  reactor::signals::action (signo=15, siginfo=0x7fd912e27bb0, ignore=<optimized out>) at core/reactor.cc:198
#2  <signal handler called>
#3  0x00007fd91b1787f3 in epoll_wait () at ../sysdeps/unix/syscall-template.S:81
#4  0x00000000006c2444 in eal_intr_thread_main ()
#5  0x00007fd91b44adc5 in start_thread (arg=0x7fd912e2e700) at pthread_create.c:308
#6  0x00007fd91b17821d in clone () at ../sysdeps/unix/sysv/linux/x86_64/clone.S:113
```

**Comment by avikivity** [comment]
Dup #601.

**Comment by avikivity** [comment]
Note: #601 was fixed post 0.16.
